src/components/AssignedUsers/AssignedUsersContainer.js

```diff
@@ -109,7 +109,7 @@ const AssignedUsersContainer = ({ permissionSetId, expanded, onToggle, tenantId
         : (
           <AssignedUsersList
             isFetching={isFetching || isMutationLoading}
-            users={users}
+            users={grantedTo.length ? users : []}
```

Comment:
looks strange, you still can enable query, but you can return empty array before loading data when array is empty

Reply:
@NikitaSedyx  What do you suggest here?

Reply:
> you still can enable query, but you can return empty array before loading data when array is empty